Directory.Build.props

```diff
@@ -38,9 +39,9 @@
     <Compile Include="**/*.cs" Exclude="$(DefaultItemExcludes)" />
   </ItemGroup>
 
-  <Target Name="DisableAnalyzers" BeforeTargets="CoreCompile" Condition="'$(Configuration)'=='Release'">
+  <Target Name="DisableAnalyzers" BeforeTargets="CoreCompile" Condition="'$(Configuration)'!='Debug-Strict'">
     <!-- Disable code style analysis on Release builds to improve compile-time performance -->
-    <ItemGroup Condition="'$(Configuration)'=='Release'">
+    <ItemGroup Condition="'$(Configuration)'!='Debug-Strict'">
```

Comment:
This Condition is inside an element with the same condition already, so do we need this?